rts/Lua/LuaOpenGLUtils.cpp

```diff
@@ -124,6 +124,9 @@ LuaMatTexture::Type LuaOpenGLUtils::GetLuaMatTextureType(const std::string& name
 		case hashString("$explosions"): { return LuaMatTexture::LUATEX_EXPLOSIONS_ATLAS; } break;
 		case hashString("$groundfx"):   { return LuaMatTexture::LUATEX_GROUNDFX_ATLAS; } break;
 
+		case hashString("$icons"): [[fallthrough]];
```

Comment:
We should add an enum for this in the future, I believe I did in some long lost work. Feel free to resolve

Reply:
you mean for various `$....` ?

Reply:
Yeah, this is exposed to the API. Hence should be documented on the API that uses it, e.g. something(texName)